src/Game/UI/uiPauseMenuDataMgr.h

```diff
@@ -236,7 +236,7 @@ class PouchItem {
     ItemUse mItemUse = ItemUse::Invalid;
     s32 mValue = 0;
     bool mEquipped = false;
-    u8 _25 = 1;
+    u8 mInInventory = 1;
```

Comment:
Seems like this should be a boolean (bool) instead of a u8.
```suggestion
    bool mInInventory = 1;
```